### PR TITLE
Re-organized code to group most of the clicast commands into a new file

### PR DESCRIPTION
**Pull request by johnpaliotta**
Reorganization of the the clicast logic into a separate file

### CODE REVIEW

**Comment by johnpaliotta** [comment]
@aytey @Zbigor should be no _real_ changes wanted to do this before adding more clicast commands (issue #31 for example)

File: src/clicast.ts [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+const fs = require("fs");
+const os = require("os");
+const path = require("path");
+const spawn = require("child_process").spawn;
+
+
+const clicastName = "clicast";
+export let clicastCommandToUse: string | undefined = undefined;
+
+const vcastqtName = "vcastqt";
```
Comment (aytey) [inline src/clicast.ts:35]: Can we call this package something else, given it isn't _just_ `clicast`?

File: src/clicast.ts [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+    else {
+      newLines.push(line);
+    }
+  }
+  fs.writeFileSync(scriptPath, newLines.join("\n"), "utf8");
+}
+
+
+const url = require("url");
+export async function loadTestScript() {
```
Comment (Zbigor) [inline src/clicast.ts:149]: We have `loadTestScript()`, `deleteTest()` and others that represent testPane interactions. We also have "low-level" `clicast` - related functions like `executeClicastCommand()`. Maybe we could leave the "low-level stuff" in this file and move the "test pane interactions" to a separate file (maybe there are some other interaction functions still in `testPane.ts` at the moment?)

File: src/utilities.ts [hunk 3]
```diff
@@ -30,15 +29,6 @@ export let globalTestInterfacePath: string | undefined = undefined;
 let globalCrc32Path: string | undefined = undefined;
 let globalPathToSupportFiles: string | undefined = undefined;
 
-export function setPaths(
```
Comment (Zbigor) [inline src/utilities.ts:33]: We use `setPaths` for unit tests, but from `pythonUtilities`. I guess it's ok to remove it from here

File: src/clicast.ts (hunk 1 above)
Comment (johnpaliotta) [reply src/clicast.ts:35]: yea, I considered that, but the only vcastqt thing is the name, will re-consider

File: src/utilities.ts (hunk 3 above)
Comment (johnpaliotta) [reply src/utilities.ts:33]: Ohh, wondered why this was not called :)  If you need this, we syould add it back into clicast.ts, want me to do that?

Comment (Zbigor) [reply src/utilities.ts:33]: I checked now. Tests use `setPaths` from `server/pythonUtilities.ts`. No need to add this back to `clicast.ts`

File: src/clicast.ts (hunk 1 above)
Comment (johnpaliotta) [reply src/clicast.ts:35]: Will change to vcastUtilities.py

Comment (johnpaliotta) [reply src/clicast.ts]: Done

File: src/clicast.ts (hunk 2 above)
Comment (johnpaliotta) [reply src/clicast.ts]: Ok I will use the convention that the commands are in testPane, and the clicast related logic is in vcastUtilities.

Let me know what you think

**Review by Zbigor — APPROVED** [review]
Up-to-date with main now and e2e tests passed

**Pull request merged**